packages/react-wildcat/src/memory.js

```diff
@@ -0,0 +1,38 @@
+const fs = require("fs");
+const memwatch = require("memwatch-next");
+
+memwatch.on("leak", function MemwatchLeak(info) {
+    fs.appendFileSync("memory-leaks.txt", `${JSON.stringify(info)}\n`);
```

Comment:
We should log this data to graylog if it's available. Also, remember to make things like filename configurable.